### DISCUSSION TITLE
Suggestion: Wireframe Models

### DISCUSSION THREAD

**Discussion by Hatchibombotar** [Mojang/minecraft-editor#119] [Feedback]
The ability to draw a line from one block to another would be useful for editor extensions, although I'm not sure if it would be difficult to implement.
![image](https://github.com/Mojang/minecraft-editor/assets/72224591/858b8697-15d5-47a9-b585-31e27ef48a61)
For example, a line tool from one block to another.
Some other ideas:
- A rotation tool, where the rotation is previewed through a wireframe cube
- A cutscene creator for maps, where the path is show with a line.
- Complex shapes or fill areas.

**Comment by conmaster2112** [comment]
> The ability to draw a line from one block to another would be useful for editor extensions, although I'm not sure if it would be difficult to implement.
> ![image](https://github.com/Mojang/minecraft-editor/assets/72224591/858b8697-15d5-47a9-b585-31e27ef48a61)
> For example, a line tool from one block to another.
> Some other ideas:
> - A rotation tool, where the rotation is previewed through a wireframe cube
> - A cutscene creator for maps, where the path is show with a line.
> - Complex shapes or fill areas.

Not hard to implement 👍
